Fixed Gradle Support

FlorianKirmaier opened the pull request:
This pull request fixes the support for Gradle.
Probably, the old version never worked properly.
With this change, cssfx should work fine with the latest version of Gradle.

It also works with javafx11 for me.

I've published it in my repository, so it can be used by everyone.
Here is a snipped, on how to use it:
```
repositories {
  jcenter()

  maven {
    url "http://sandec.bintray.com/repo"
  }
}
dependencies {
  compile 'de.sandec.cssfx:cssfx:1.0.4'
}
```

McFoggy commented:
thanks for the contribution.
This behavior has changed between gradle 3.x & gradle 4.x (see [here](https://discuss.gradle.org/t/build-path-for-java-classes-changed-from-3-x-to-4-x/24168)).

we should keep 3.x in the loop ; I'll enhance your commit in that direction

> I've published it in my repository, so it can be used by everyone.
hum not sure it is that a good idea ; let's take the time to fix it upstream please

FlorianKirmaier commented:
Thank you for the fast response!

The main reason why I've published it on my respository was, because the last change is 5 years old, so i thought the repository might be dead.

About the support for 3.x. But i think it never really worked because of a copy-paste error.
On line 81 and 85, there shold be the same string which wasn't the case.

I've also added two further commits.
It now works also, when starting the program from IntelliJ. (at least on a gradle project)
The css now updated faster on mac. Before it was 5 seconds, now it's 1 second.
